lib/src/widgets/auth_card.dart

```diff
@@ -527,7 +532,7 @@ class _LoginCardState extends State<_LoginCard> with TickerProviderStateMixin {
         }
       },
       validator: widget.passwordValidator,
-      onSaved: (value) => _authData['password'] = value,
+      onSaved: (value) => auth.password = value,
```

Comment:
Will update this and similar lines. Can't use passed in auth, need to get it from provider here in onSaved, else it will become disposed.

Reply:
Actually, this was ok as it was, since we're listening for auth updates from Provider. I removed some extraneous code and added auth as param in other places for consistency.